master/internal/rm/kubernetesrm/informer_intg_test.go

```diff
@@ -213,12 +180,165 @@ func TestNodeInformer(t *testing.T) {
 					Object: node,
 				}
 			}
+
+			wg.Wait()
+			assert.Equal(t, tt.output, currNodes)
+		})
+	}
+}
+
+func TestEventListener(t *testing.T) {
+	cases := []struct {
+		name       string
+		expected   error
+		eventNames []string
+	}{
+		{"zero events", nil, []string{}},
+		{"listener success", nil, []string{"A"}},
+		{
+			"listener success & event ordering success",
+			nil,
+			[]string{"A", "B", "C", "D", "E"},
+		},
+		{
+			"listener success & event ordering failure",
+			nil,
+			[]string{"A", "B", "C", "D", "E"},
+		},
```

Comment:
these cases are the same?

Reply:
yeah, I was in the middle of refactoring them -- next commit will have the proper failures included